app/helpers.php

```diff
@@ -196,6 +196,15 @@ function read_image_properties_from_string($string)
     }
 }
 
+function require_login($text_key, $link_text_key)
```

Comment:
Could make this accept a single parameter and do the `_` and `link_text` lookups implicitly, but that might not be obvious when adding translations...